### PR TITLE
hello: add page

### PR DESCRIPTION
**Pull request by sahilister**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/hello.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+`hello`
+
+- Print "hello, world", the traditional type:
+
+`hello --traditional`
+
+- Print a custom text:
+
+`hello --greeting="{{tldr}}"`
```
Comment (schneiderl) [inline pages/linux/hello.md:16]: I don't think I like this parameter name. We usually try to give more descriptive names. I would recommend something like "Some greeting".

**Review by schneiderl — CHANGES_REQUESTED** [review]
Hi,
I've just left a minor comment for you to take a look at. 
Thank you for your contribution.

File: pages/linux/hello.md (hunk 1 above)
Comment (sahilister) [reply pages/linux/hello.md:16]: Taking a cue from [echo](https://github.com/tldr-pages/tldr/blob/master/pages/common/echo.md), how about, `Print a text message` ?

Using "greeting.." might be ambiguous, I believe. (even though it's the argument itself :D )

Comment (schneiderl) [reply pages/linux/hello.md:16]: Yes that is great 👍

Comment (sahilister) [reply pages/linux/hello.md:16]: Updated.

**Review by einverne — APPROVED** [review]
Looks good. Thanks @sahilister

File: pages/linux/hello.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/hello.md:16]: I'm not seeing the update, @sahilister?

Comment (sahilister) [reply pages/linux/hello.md:16]: Hello @sbrl , [here](https://github.com/tldr-pages/tldr/pull/4586/commits/bb1bda182083de3685ae09d4778329471f207ff8) it's showing changes from `Print a custom text` -> `Print a text message`.

A tiny change, maybe that's why it slipped. :)

**Comment by schneiderl** [comment]
@sahilister any updates on the pending review?

File: pages/linux/hello.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/hello.md:16]: Oh I see @sahilister! Thanks for that. I think @schneiderl also meant to change the `tldr` message here too. What about this?
```suggestion
`hello --greeting="{{greeting_text}}"`
```

Comment (sahilister) [reply pages/linux/hello.md:16]: aah! sorry I missed that one, thanks @sbrl

**Comment by sahilister** [comment]
@schneiderl I misunderstood your suggestion earlier which has be corrected, thanks to @sbrl . Please have a look again :)

**Comment by schneiderl** [comment]
No problem @sahilister, we made it work after all. 
Looks great to me. Thank you.

**Review by schneiderl — APPROVED** [review]

**Pull request merged**